packages/bitcore-wallet-service/src/lib/chain/btc/index.ts

```diff
@@ -239,19 +238,17 @@ export class BtcChain implements IChain {
         break;
       default:
         scriptSize = 34;
-        logger.warn('Unknown address type at getEstimatedSizeForSingleOutput:', addressType);
+        // logger.warn('Unknown address type at getEstimatedSizeForSingleOutput:', addressType);
```

Comment:
This is commented. It's okay?

Reply:
yes. this happens at the begining of the estimation. addresses are not yet defined. no need to warn.